packages/dds/tree/README.md

```diff
@@ -11,15 +11,10 @@ and some of the implications of those goals for developers working on this packa
 
 Notable consideration that early adopters should be wary of:
 
--   The persisted format is not stable and long term support for persisted format is not yet in effect.
-    This means clients running more recent versions of SharedTree may find themselves unable to load old document or may corrupt them when editing.
-    Current estimate for LTS is April 2024.
--   SharedTree currently has unbounded memory growth:
-    -   Removed content is retained forever.
-    -   Accessing an `EditableField` object (from its parent, e.g., with `getField`) in a loop will leak unbounded memory.
--   All changes are atomized by the `visitDelta` function.
-    This means that, when inserting/removing/moving 2 contiguous nodes, the `visitDelta` function will call the `DeltaVisitor` twice (once for each node) instead of once for both nodes.
-    Change notification consumers that are downstream from the `DeltaVisitor` will therefore also see those changes as atomized.
+-   The persisted format is now stable: documents created with released versions 2.0.0 of greater of this package will be supported long term.
+-   All range changes are atomized.
+    This means that, when inserting/removing/moving multiple contiguous nodes the edit is split up into separate single node edits.
+    This can impact the merge behavior of these edits, as well as the performance of large array edits.
 -   Some documentation (such as this readme and [the roadmap](docs/roadmap.md)) are out of date.
```

Comment:
If this is correct, surely we should update this readme, and not say it is out of date?

Reply:
there are plenty of out of date things further below.